src/Components/App/App.js

```diff
@@ -33,7 +33,7 @@ class App extends Component {
       <>
       <article className='App'>
         <Header />
-        {this.state.error && <h3 className='error-msg'>{this.state.error}</h3>}
+        {this.state.error && <h3>{this.state.error}</h3>}
```

Comment:
Lets change this back to having a className with an error-msg for cypress test passing purposes.  

I think this is an issue between `atom` `terminal` and `git` Not all of our changes are pushing up.